gnovm/cmd/gno/test.go

```diff
@@ -394,6 +410,18 @@ func runTestFiles(
 		if rep.Output != "" && (verbose || rep.Failed) {
 			io.ErrPrintfln("output: %s", rep.Output)
 		}
+
+		if printRuntimeMetrics {
+			imports := m.Store.NumMemPackages() - numPackagesBefore - 1
+			// XXX: pretty print big numbers
+			// XXX: store changes
+			// XXX: total alloc
+			// XXX: max mem consumption
+			io.ErrPrintfln("---       runtime: cycle=%dk imports=%d allocs=TODO memory=TODO store=TODO",
```

Comment:
`gas` is mentioned in the flag description but it's not in the output.

Reply:
Yeah, will add it.

I'm unsure about the format. Should I keep it as is or add more details to the existing execution time? I prefer an idiomatic format, even if it's not typical for Gophers.

Reply:
anything to get started.